configs/peer/config.json

```diff
@@ -7,7 +7,7 @@
     "TRUSTED_PEERS": [
       {
         "address": "127.0.0.1:1337",
-        "public_key": "ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0"
+        "public_key": "ed01201c61faf8fe94e253b93114240394f79a607b7fa55f9e5a41ebec74b88055768b"
```

Comment:
When this gets merged, you need to notify all the library developers about this change. 

I would also recommend not making this change and using the old key instead.

Reply:
I am in favor of this change because it clearly delineates between 3 pairs of keys - genesis key, peer key and client/account key